tasks/wait-for-node-ready.yml

```diff
@@ -23,17 +23,18 @@
   register: node_output
   delegate_to: "{{ openshift_oc_tools_host }}"
   until:
-  - node_output.results is defined
-  - node_output.results.returncode is defined
-  - node_output.results.returncode == 0
-  - node_output.results.results is defined
-  - node_output.results.results | length > 0
-  - node_output.results.results[0].status is defined
-  - node_output.results.results[0].status.conditions is defined
-  - node_output.results.results[0].status.conditions | selectattr('type', 'match', '^Ready$') | map(attribute='status') | join | bool == True
+  - node_output.module_results is defined
+  - node_output.module_results.returncode is defined
+  - node_output.module_results.returncode == 0
+  - node_output.module_results.results is defined
+  - node_output.module_results.results | length > 0
+  - node_output.module_results.results[0].status is defined
+  - node_output.module_results.results[0].status.conditions is defined
+  - node_output.module_results.results[0].status.conditions | selectattr('type', 'match', '^Ready$') | map(attribute='status') | join | bool == True
   # Give the node three minutes to come back online.
   retries: 36
   delay: 5
+  when: inventory_hostname not in groups['masters']
```

Comment:
Why was this condition added? Masters should be in a ready state too.

Reply:
Good catch. This should not have been added. Removed.